<?xml version="1.0"?>
<launch>
    <!-- Sets the obstacle representation type, accepted types are: "ellipse", "wedge","circles", "hybrid_circle", and "hybrid_ellipse" (full list in getObstacles())-->
    <arg name="obstacle_type" default="hybrid_circle" />
    <param name="obstacle_type" type="string" value="$(arg obstacle_type)" />

    <!-- Set if screenshots should be taken or not -->
    <arg name="screenshot" default="false" />
    <param name="screenshot" type="bool" value="$(arg screenshot)" />

    <!-- Set if pathfinding problem should be plotted or not -->
    <arg name="plot_pathfinding_problem" default="false" />
    <param name="plot_pathfinding_problem" type="bool" value="$(arg plot_pathfinding_problem)" />

    <!-- Sets planner type, accepted types include: "RRTStar", "RRTSharp","RRTXStatic" (full list in allocatePlanner()) -->
    <arg name="planner_type" default="RRTStar" />
    <param name="planner_type" type="string" value="$(arg planner_type)" />

    <node pkg="local_pathfinding" type="main_loop.py" name="main_loop" output="screen"/>
    <node pkg="local_pathfinding" type="ros_interface.py" name="ros_interface" output="screen"/>
    <node pkg="local_pathfinding" type="global_wind.py" name="global_wind" output="screen"/>
</launch>

